### PR TITLE
Support config row definition

### PR DESCRIPTION
**Pull request by ondrajodas**
Solving https://github.com/keboola/db-writer-common/issues/31

### CODE REVIEW

File: src/MSSQL/Configuration/MssqlActionConfigRowDefinition.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class MssqlActionConfigRowDefinition implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('parameters');
+        $rootNode = $treeBuilder->getRootNode();
+
+        $rootNode
```
Comment (pivnicek) [inline src/MSSQL/Configuration/MssqlActionConfigRowDefinition.php:18]: ignoreExtraKeys please:)

File: tests/data/testConnection/config.json [hunk 2]
```diff
@@ -9,7 +9,6 @@
       "database": "test",
       "user": "sa",
       "#password": "yourStrong(!)Password"
-    },
-    "tables": []
```
Comment (pivnicek) [inline tests/data/testConnection/config.json:13]: asi lepsi pouzivat novy dir `rowTestConnection` a nechat tenhle config jak to je

**Review by pivnicek — CHANGES_REQUESTED** [review]
dobry, jen maly vec s testem

File: tests/data/testConnection/config.json (hunk 2 above)
Comment (ondrajodas) [reply tests/data/testConnection/config.json:13]: nice catch :)

**Review by pivnicek — APPROVED** [review]
lgtm

**Pull request merged**